packages/sdk/client/src/services/worker-client-services.ts

```diff
@@ -0,0 +1,170 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { Event, Trigger, synchronized } from '@dxos/async';
+import {
+  type ClientServices,
+  type ClientServicesProvider,
+  clientServiceBundle,
+  DEFAULT_CLIENT_CHANNEL,
+} from '@dxos/client-protocol';
+import { WorkerProxyRuntime } from '@dxos/client-services';
```

Comment:
it would be beneficial for the bundle size to do a dynamic import in the `open` method for client-services package